programs/gateway/src/instructions/update_voter_weight_record.rs

```diff
@@ -41,6 +50,8 @@ pub fn update_voter_weight_record(
         &ctx.accounts.registrar.gatekeeper_network,
         None
     ).or(Err(error!(GatewayError::InvalidGatewayToken)))?;
+    
+    let voter_weight = extract_input_voter_weight(&ctx.accounts.input_voting_weight.to_account_info(), &ctx.accounts.registrar);
```

Comment:
If there there is no previous plugin then let's explicitly set all fields on `VoteWeightRecord` to avoid stale state (ex. copied from another plugin when it was enabled and then disabled)